### ISSUE TITLE
Provide a way to stop it

### ISSUE THREAD

**Issue by mcasimir** [Beh01der/node-docker-monitor#2]
Awesome lib.

It would be very useful to have a clean way to stop it, especially for integration testing.

Without much changes in the interface could be something like this:

``` js
 let stopMonitor = monitor(callbacks);
```

a bit cleaner but requires breaking changes:

``` js
let monitor = new Monitor(dockerConfig);

monitor.on('container:start', fn);
monitor.on('container:stop', fn);

monitor.start();
monitor.stop();
```

**Comment by Beh01der** [comment]
I've implemented it in 1.0.10

``` javascript
let m = monitor(callbacks);
m.stop();
```

I've also added 2 extra callback methods:
- onMonitorStarted
- onMonitorStopped

To start monitoring after stopping, you need to call **monitor** again

``` javascript
let m = monitor(callbacks);
```

**Comment by mcasimir** [comment]
Awesome, thanks

Il gio 28 lug 2016, 08:52 Andrey Chausenko notifications@github.com ha
scritto:
